backend/app/Infrastructure/Persistence/Repositories/PredisIssueRepository.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Repositories;
+
+use App\Domain\Entities\Issue;
+use App\Domain\Repositories\IssueRepository;
+use Predis\Client;
+
+class PredisIssueRepository implements IssueRepository
+{
+    private Client $client;
+
+    public function __construct()
+    {
+        $this->client = new Client([
+            'scheme' => $_ENV['REDIS_SCHEME'],
```

Comment:
usar `getenv()`

Reply:
Vos sabes que justamente lei que en https://packagist.org/packages/vlucas/phpdotenv se aclara que no hay que usar getenv ni putenv porque son inseguras y pueden causar problemas en threads
`Using getenv() and putenv() is strongly discouraged due to the fact that these functions are not thread safe`
De todas formas lo configure para que se use getenv porque usar la variable mágica _ENV tampoco es que sea la mejor práctica no?